notebooks/DrizzlePac/align_to_catalogs/align_to_catalogs.ipynb

```diff
@@ -4,42 +4,80 @@
    "cell_type": "markdown",
```

Comment:
<p>There is more straightforward way to get the basename:</p>

<pre class="ql-syntax" spellcheck="false">	flc_name = os.path.basename(flc)
</pre><p>It's safer to check if the directory exists before attempting to remove it.</p>

<pre class="ql-syntax" spellcheck="false">if os.path.exists('mastDownload'):
	shutil.rmtree('mastDownload')
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/265/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Agreed, I have implemented this change.